cleanup-cmssdt

```diff
@@ -51,6 +51,7 @@ DIRS="lizard flawfinder invalid-includes cmssw-afs-eos-comparison ubsan_logs ib-
 DIRS="${DIRS} check_headers valgrind HLT-Validation ib-static-analysis ib-baseline-tests ib-dqm-tests profiling igprof"
 DIRS="${DIRS} iwyu material-budget das_query build-any-ib check-unused-cmsdist-packages class_versions"
 DIRS="${DIRS} test-os-alma8 test-os-cs8 test-os-ubi8 test-os-lxplus8 test-os-rhel8 test-os-rocky8 test-os-el8 cms-containers-run-cmssw-test"
+DIRS="${DIRS} baseLineComparisonsCUDA baseLineComparisonsROCM"
```

Comment:
@iarspider , why this was needed? I think `dir=baseLineComparisons${type}` at line 29 should take care of this cleanup ... right?

Reply:
Fixed